packages/objectloader/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "lint": "eslint . --ext .js,.ts",
     "build:dev": "rollup --config",
-    "build": "NODE_ENV=production rollup --config",
+    "build": "set NODE_ENV=production && rollup --config",
```

Comment:
This is a revert to the original state

Reply:
The package lock has probably changed because I also removed two libraries which I initially added, but removed since they are no longer needed anymore.https://github.com/specklesystems/speckle-server/commit/78b54a6594cd8f25bde54e2aee9225f4e6d330d3